tsconfig.json

```diff
@@ -1,12 +1,8 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "lib": [
-      "es2017",
-      "es7",
-      "es6"
-    ],
-    "allowSyntheticDefaultImports": false,
+    "target": "es5",
```

Comment:
Sure - `target` refers to what version of Javascript that typescript should seek to compile for. The previous value, `esnext`, is a kind of catch all for "The most recent version of JS that Typescript supports" - a little analogous to setting a gem dependency to "HEAD", it keeps us "up to date" at the risk of making things less predictable. 

Since this is about the end result of typescript compilation, there's no real danger setting it to a value that is a little more generic, unless we wanted it to natively take advantage of some weird modern feature that there existed no built-in polyfill for. 

(The following key, "lib", is another layer of compatibility targets, but is generally redundant next to `target`.)